src/components/item-categories/index.js

```diff
@@ -1,10 +1,31 @@
 import React from 'react';
-import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Badge } from 'react-rainbow-components';
 
-const Styled = styled.div``;
+const ItemCategories = props => {
+    const { className, style, categories } = props;
 
-const ItemCategories = () => {
-    return <Styled>ItemCategories</Styled>;
+    if (categories) {
+        return (
+            <div className={className} style={style}>
+                {categories.map(category => (
+                    <Badge label={category} key={category} />
```

Comment:
I know that each category should be different but in order to pass a more robust key and to make sure it will be unique we should construct it attaching the index like:
```const key = `${category}-${index}`;```